### ISSUE TITLE
Wysyłam z Allegro.cz Inpost: Przesyłka nie została zamówiona

### ISSUE THREAD

**Issue by lukasnet** [allegro/allegro-api#10042]
Występuje problem: 
 - [ x] z zasobem REST API / with a REST API resource
 - [ x] ze środowiskiem produkcyjnym / with the production environment
 - [ ] ze środowiskiem testowym / with the test environment
 - [ ] z logowaniem lub autoryzacją / with login or authorization
 - [ ] na portalu https://developer.allegro.pl/ / at https://developer.allegro.pl/en/
 - [ ] inny / other

Od zeszłego tygodnia występują cały czas problemy z nadawaniem paczek na Allegro.cz przez Wysyłam z Allegro do InPost. Zarówno tworzenie paczki starą metodą _parcel-management_  jak i nową _shipment-management_. Proszę o weryfikację gdzie leży problem.
CommandID: 57e4894a-0af4-458c-9ace-d79430d8c88a
CommandID: 14290b2a-96a4-46a9-bac1-0a53f38cf377

**ERROR: Przesyłka nie została zamówiona. Przekroczono czas oczekiwania na potwierdzenie zlecenia ze strony przewoźnika. Ponów operację później.**

**Comment by MaciejFrackowiak** [comment]
Tak, w przypadku tej usługi Allegro International trwają prace po stronie operatora by wyeliminować problem. Przepraszam za utrudnienia.

**Comment by MaciejFrackowiak** [comment]
Problem generuje numer telefonu kupującego - musi zaczynać się na 6 lub 7, mieć 9 cyfr, ew. czeski prefiks +420. W podanym przykładzie widziałem 0 na początku ;)
